ccjjmartin/aquifer-inope
# Challenge3 - Retrieving and displaying values

This PR seeks to:
- Create a table presentation of the data.
- Add a table sort to the data.
- Allow users to change their answer.
- Add indicators alerting the user they need to respond to the policy.
- Limit responses to only the current user on the current node.
## Steps to test involve:
- Run update.php on sites that have already been running the module.
  - To test that update.php worked properly your boolean values for note_status should now be inverted.
- Visit a new node:
  - Check that the Review of @node_title now has (NEW) at the end of the title.
  - Check that a status message alerts the user that they need to respond to the policy.
- As an administrator make an update to a node and create a new revision:
  - Check that the Review of @node_title now has (UPDATED) at the end of the title.
  - Check that a status message alerts the user that they need to update their response to the policy.
- After you have gone through at least two revisions:
  - Visit the Review of @node_title tab and verify that a table is appearing.
  - Verify that you have the proper number of responses for only your user.
  - Verify that no other node's responses are appearing in the table.
- Once you have verified that the responses are appearing properly:
  - Verify that the default order of the data is DESC by date.
  - If you already reordered the data remove any query parameters at the end of the URL.
  - Try reordering the data by clicking on the headers.
- Great!  We are almost there.
  - Click the edit button next to a previously submitted response.
  - Change your response.
  - Verify that the response was updated.
